core/src/main/java/discord4j/core/GatewayDiscordClient.java

```diff
@@ -213,6 +211,59 @@ public RestClient getRestClient() {
                 .map(data -> new Webhook(this, data));
     }
 
+    /**
+     * Executes this webhook without waiting for a confirmation that a message was created.
+     *
+     * @param webhookId The id of the webhook.
+     * @param webhookToken The token of the webhook.
+     * @param spec A {@link Consumer} that provides a "blank" {@link discord4j.core.spec.WebhookExecuteSpec} to be
+     * operated on.
+     * @return A {@link Mono} where, upon successful webhook execution, completes. If the message fails to save,
+     * an error IS NOT emitted through the {@code Mono}.
+     */
+    public Mono<Void> executeWebhookById(Snowflake webhookId, String webhookToken,
```

Comment:
Why are these methods necessary when Webhook#execute() exists? Is it to do with the need of the token?

Reply:
removed methods